src/mixins/selection.js

```diff
@@ -0,0 +1,42 @@
+// @flow
+
+/**
+ * CSS to style the selection psuedo-element.
+ *
+ * @example
+ * // Styles as object usage
+ * const styles = {
+ *   ...selection({
+ *     'background': 'blue'
+ *   }, 'section')
+ * }
+ *
+ * // styled-components usage
+ * const div = styled.div`
+ *   ${selection({'background': 'blue'}, 'section')}
```

Comment:
My flow-fu isn't super strong. Setting `styles` to `Object|Array` throws this error `Incorrect number of type parameters (expected 1)`. Is there a specific syntax beyond `any` when you want a param to be one of two types?